src/Package/Test/DataInspect/ReplCommandsTest.cs

```diff
@@ -0,0 +1,232 @@
+    public class ReplCommandsTest {
+        private readonly IRSessionProvider _sessionProvider;
+        private readonly IObjectDetailsViewerAggregator _aggregator;
+
+        public ReplCommandsTest() {
+            _sessionProvider = VsAppShell.Current.ExportProvider.GetExportedValue<IRSessionProvider>();
+            _aggregator = VsAppShell.Current.ExportProvider.GetExportedValue<IObjectDetailsViewerAggregator>();
+        }
+
+        [Test]
+        [Category.Variable.Explorer]
+        public async Task ViewLibraryTest() {
+            var cb = Substitute.For<IRSessionCallback>();
+            cb.ViewLibrary().Returns(Task.CompletedTask);
+            using (var hostScript = new RHostScript(_sessionProvider, cb)) {
+                using (var inter = await hostScript.Session.BeginInteractionAsync()) {
+                    await inter.RespondAsync("library()" + Environment.NewLine);
+                }
+            }
+            await cb.Received().ViewLibrary();
+        }
+
+        [Test]
+        [Category.Variable.Explorer]
+        public async Task ViewDataTest01() {
+            var cb = Substitute.For<IRSessionCallback>();
+            cb.When(x => x.ViewObject(Arg.Any<string>(), Arg.Any<string>())).Do(x => { });
+            using (var hostScript = new RHostScript(_sessionProvider, cb)) {
+                using (var inter = await hostScript.Session.BeginInteractionAsync()) {
+                    await inter.RespondAsync("View(mtcars)" + Environment.NewLine);
+                }
+            }
+            cb.Received().ViewObject("mtcars", "");
+        }
+
+        [Test]
+        [Category.Variable.Explorer]
+        public async Task ViewerExportTest() {
+            using (var hostScript = new RHostScript(_sessionProvider)) {
+                var funcViewer = await _aggregator.GetViewer("lm");
+                funcViewer.Should().NotBeNull().And.BeOfType<FunctionViewer>();
+
+                var gridViewer = await _aggregator.GetViewer("airmiles");
+                gridViewer.Should().NotBeNull().And.BeOfType<Viewer1D>();
+
+                gridViewer = await _aggregator.GetViewer("mtcars");
+                gridViewer.Should().NotBeNull().And.BeOfType<TableViewer>();
+
+                gridViewer = await _aggregator.GetViewer("AirPassengers");
+                gridViewer.Should().NotBeNull().And.BeOfType<Viewer1D>();
+
+                gridViewer = await _aggregator.GetViewer("list(c(1:10))");
+                gridViewer.Should().NotBeNull().And.BeOfType<ListViewer>();
+
+                gridViewer = await _aggregator.GetViewer("c(1:10)");
+                gridViewer.Should().NotBeNull().And.BeOfType<VectorViewer>();
+
+                gridViewer.Capabilities.Should().HaveFlag(ViewerCapabilities.List | ViewerCapabilities.Table);
+            }
+        }
+
+        [Test]
+        [Category.Variable.Explorer]
+        public void ViewerTest02() {
```

Comment:
I don't understand what this test covers that the `ViewerExportTest` doesn't cover